src/Core/Settings.h

```diff
@@ -156,7 +156,7 @@ struct Settings : public SettingsCollection<Settings>
     M(SettingUInt64, priority, 0, "Priority of the query. 1 - the highest, higher value - lower priority; 0 - do not use priorities.", 0) \
     M(SettingInt64, os_thread_priority, 0, "If non zero - set corresponding 'nice' value for query processing threads. Can be used to adjust query priority for OS scheduler.", 0) \
     \
-    M(SettingBool, log_queries, 0, "Log requests and write the log to the system table.", 0) \
+    M(SettingBool, log_queries, 1, "Log requests and write the log to the system table.", 0) \
```

Comment:
Just out of curiosity what is the reason for to enable it by default?
Maybe worth enabling only for errors? I.e. `log_queries_min_type='EXCEPTION_BEFORE_START'` ?

Reply:
For two reasons:

1. Every time we provide consultancy for users, the first question from us is "do you have query log enabled?". It is crucial for investigations. Better to have it by default to avoid obstacles.

2. The reason why it was not enabled by default it the following: it will write data to disk and use disk space even if the server is readonly. It can be surprising for users. But now we already have metric_log and trace_log enabled by default.